includes/class-wc-gateway-swedbank-pay-cc.php

```diff
@@ -845,8 +845,10 @@ public function return_handler() {
 			$order_id  = absint(  wc_clean( $_GET['order_id'] ) ); // WPCS: input var ok, CSRF ok.
 			$order_key = empty( $_GET['key'] ) ? '' : wc_clean( wp_unslash( $_GET['key'] ) ); // WPCS: input var ok, CSRF ok.
 
-			if ( empty( $order_id ) || empty( $order_id ) ) {
-				throw new Exception( 'An order ID or order key wasn\'t provided' );
+			if ( empty( $order_id ) || empty( $order_key ) ) {
+				if ( ! in_array( $_SERVER['REMOTE_ADDR'], [ '82.115.146.1' ] ) ) {
```

Comment:
Didn't we agree to ignore all requests coming from IP `82.115.146.1`? This logic only applies if there is no `$order_id`. Please move the check for `82.115.146.1` to the outer scope, assign the IP address to a variable clearly explaining what the IP address represents (such as `$payex_origin_ip_address`) and return from the method if the IP address is equal to it.

Reply:
@olegisk, @aait, please follow up.

Reply:
@asbjornu Done